js/ssai_pulse.js

```diff
@@ -144,22 +144,33 @@ OO.Ads.manager(function(_, $)
 
       if (adManagerMetadata)
       {
-        if (adManagerMetadata["cacheBuster"])
+        // allow boolean true/false
+        if (_.isBoolean(adManagerMetadata["cacheBuster"]))
         {
-          if (adManagerMetadata["cacheBuster"] === "true")
+          if (adManagerMetadata["cacheBuster"])
```

Comment:
If we've determined that adManagerMetadata["cacheBuster"] is a boolean, can we set this.bustTheCache to this?
this.bustTheCache = adManagerMetadata["cacheBuster"];